### DISCUSSION TITLE
Android?

### DISCUSSION THREAD

**Discussion by wolbar** [pybricks/support#388] [General]
Hi, anyone try it on an Android tablet yet?

**Comment by dlech** [comment]
We haven't focused on Android, so it might not look great, but it should work.

**Reply by wolbar** [reply]
Ok, I'll try it ;). It would be nice to have both Powered Up and Pybricks code on the same device, with BT keyboards. I teach robotics and having multiple systems is a pain. Young kids only swipe these days. Older kids can do both so it would be great to draw a progarm in Powered Up and then write it in Python. Or maybe I'm wrong but I'll try it. My favorite comic is a boy with a shocked look on his face while holding a book and exclaiming: "Father, the page won't swipe!". So true...

**Reply by laurensvalk** [reply]
If you try it, let us know how it goes :) We are always interested in feedback to make it better.

And with an external BT keyboard, it might be just fine.

**Reply by icefloe01** [reply]
I have tried to connect to a Move Hub using Chrome on a Samsung S10+ without success. While I can see the hub in my phone's Bluetooth settings, it does not show up in the pybricks webpage.

**Reply by laurensvalk** [reply]
What is the name of the hub when you see it in settings?

When you follow these steps, at which point does it no longer work? https://youtu.be/QLL7EegBGr0

**Reply by icefloe01** [reply]
Oh thank you! THAT worked. That's apparently the one video I missed. 🤣 Again, thank you very much!

**Reply by laurensvalk** [reply]
Glad it worked!

But that probably means we have to improve our other videos. Which other videos did you see?

**Reply by icefloe01** [reply]
I doubt you need to improve anything, I just need to slow down.🤣 I can end jumping ahead too fast. I was trying to connect by Bluetooth first because I was thinking I needed to do that before being able to update the firmware.
